code/modules/mob/living/carbon/human/human_attackhand.dm

```diff
@@ -311,7 +311,7 @@
 			else
 				playsound(loc, 'sound/weapons/slice.ogg', 25, 1, -1)
 
-			visible_message("<span class='danger'>[M] has [M.species.attack_verb]ed [src]!</span>")
+			visible_message("<span class='danger'>[M] has [M.species.attack_verb]med [src]!</span>")
```

Comment:
under this line of thought I'd be seeing slashmed and tacklemed but under testing I'm not so I don't think it requires much more.